xds/src/test/java/io/grpc/xds/RouteMatchTest.java

```diff
@@ -0,0 +1,166 @@
+/*
+ * Copyright 2020 The gRPC Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.grpc.xds;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import com.google.re2j.Pattern;
+import io.grpc.xds.RouteMatch.FractionMatcher;
+import io.grpc.xds.RouteMatch.HeaderMatcher;
+import io.grpc.xds.RouteMatch.HeaderMatcher.Range;
+import io.grpc.xds.RouteMatch.PathMatcher;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+import org.junit.Before;
+import org.junit.Test;
+
+/** Tests for {@link RouteMatch}. */
+public class RouteMatchTest {
+
+  private final Map<String, Set<String>> headers = new HashMap<>();
+
+  @Before
+  public void setUp() {
+    headers.put("content-type", new HashSet<>(Collections.singletonList("application/grpc")));
```

Comment:
Why not just use `Set<T> Collections.singleton(T o)`

Reply:
Yep, good to know.